app/controllers/alaveteli_pro/draft_info_requests_controller.rb

```diff
@@ -0,0 +1,121 @@
+# -*- encoding : utf-8 -*-
+# app/controllers/alaveteli_pro/request_controller.rb
+# Dashboard controller, for pro user dashboards.
+#
+# Copyright (c) 2008 UK Citizens Online Democracy. All rights reserved.
+# Email: hello@mysociety.org; WWW: http://www.mysociety.org/
+
+class AlaveteliPro::DraftInfoRequestsController < AlaveteliPro::BaseController
+  def new
+    @draft_info_request = DraftInfoRequest.new
+  end
+
+  def edit
+    @draft_info_request = DraftInfoRequest.find(params[:id])
```

Comment:
Should scope to `current_user`, otherwise you can just bash in ID numbers to the URL and load other users' drafts:

```ruby
@draft_info_request = current_user.draft_info_requests.find(params[:id])
```

Should do this for all actions